### PR TITLE
docs: Fix links to Markdown files

### PR DESCRIPTION
**Pull request by cblichmann**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -25,19 +25,19 @@ This project is composed internally of several internal components, depicted in
 ## Further reading
 
 * [Glossary](./glossary)
```
Comment (jprendes) [inline docs/README.md:27]:
```suggestion
* [Glossary](./glossary.md)
```

**Review by jprendes — COMMENTED** [review]
Hi @cblichmann, thanks for your contribution!
CI is failing because the commit is not signed off.
Could you sign off the commit?

File: docs/README.md (hunk 1 above)
Comment (Real-K5) [reply docs/README.md:27]: goodz

**Comment by Real-K5** [comment]
i think it is a good projects, give me more value

**Comment by danbugs** [comment]
I believe this closes #59

**Comment by danbugs** [comment]
This PR got superseded by #60 and #68 , so I'm closing.

**Pull request closed without merging**